docs/tests/conditionals.rst

```diff
@@ -0,0 +1,209 @@
+Conditional  Statements
+=======================
+
+The conditional section of the pavilion config consists of two main
+directives, ``only_if`` and ``not_if``. This documentation will cover
+how to use them as well as details associated with their use in test
+configs.
+
+Why conditional statements? As Pavilion becomes more popularized as an
+HPC test harness, the test suites it runs will theoretically grow. It is
+unrealistic to manage unique test suites for specific machines. By using
+conditional statements users can have one large test suite that can run
+across all systems. If a user has a test that is specific to a unique
+computer architecture, they can simply add 'only_if arch is unique-arch'.
+Conditional Statements can greatly simplify your testing procedure.
+
+
+only_if
+~~~~~~~
+
+The ``only_if`` section of the pavilion test config allow users to
+specify a series of conditions. As the name would suggest, the test
+will run if and only if ALL conditions are met.
+
+The syntax for writing an ``only_if`` condition is fairly
+straightforward. The ``only_if`` directive is followed by a
+dictionary that should contain at least one key. The key(s)
+should consist of a variable(s) that pavilion can resolve. Below is
+a basic example.
+
+.. code:: yaml
+
+    basic_test:
+        only_if:
+            user: ['calvin']
+        run:
+            cmds:
+                ...
+
+In this example the dictionary following ``only_if`` has a key
+``user`` and the value of 'calvin'. Pavilion will resolve the key and
+check who the actual user is. Pavilion will then compare the resolved
+key with the value and if ``user == calvin`` then the test will run as
+``calvin == calvin``.
+
+Conditional statements can also parse dictionaries with multiple keys
+and values, let's look at a more complicated test.
+
+.. code:: yaml
+
+    advanced_test:
+        only_if:
+            user: ['calvin', 'paul']
+            sys_os: ['linux']
+            weekday: ['Monday', 'Wednesday', 'Friday']
+        run:
+            cmds:
+                ...
+
+In this example we have three keys that need to be resolved: [user,
+sys_os, weekday]. As with the simpler example above, each key is
+resolved and checked with the values supplied to it. An important
+thing to remember is ``only_if`` needs to have a match on ALL keys.
+If the user is paul testing on a linux machine, but it's Tuesday,
+the test will not run.
+
+not_if
+~~~~~~
+
+The ``not_if`` section of the pavilion test config allow users to
+specify when a test should be skipped. As the name suggests, a test
+will be skipped if anything matching the ``not_if`` conditional
+is found.
+
+The syntax for a ``not_if`` conditional statement is straightforward.
+A user will specify the directive ``not_if`` followed by a dictionary
+with at least one key in their pavilion test config. The key(s)
+should consist of a variable(s) that pavilion can resolve. Below
+is a basic example.
```

Comment:
You describe this is such different language that it seems like an entirely different concept. Instead, refer back to how 'only_if' conditions work, and just state how they're different.